catalog/catalog.jsonnet

```diff
@@ -119,6 +119,7 @@ local self_url = base_url + base_filename;
     ee.link.child_catalog('WWF', base_url),
     ee.link.child_catalog('WorldPop', base_url),
     ee.link.child_catalog('YALE', base_url),
+    ee.link.child_catalog('edf-methanesat-ee', base_url),    
```

Comment:
@simonff is this the file and link you mean, or something else?

Reply:
Yes, thank you. Sorry if I missed it originally.